modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/contestable_issues_controller.rb

```diff
@@ -7,6 +7,7 @@
 module AppealsApi::V2
   module DecisionReviews
     class ContestableIssuesController < AppealsApi::ApplicationController
+      FORM_NUMBER = 'CONTESTABLE_ISSUES_HEADERS'
```

Comment:
See other comment about `FORM_NUMBER`, which was originally defined in the segmented controller.